Tests/SecureXPCTests/XPC Codable round trip tests.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  XPC Codable round trip tests.swift
+//  
+//
+//  Created by Alexander Momchilov on 2021-11-07.
+//
+
+import XCTest
+@testable import SecureXPC
+
+class SampleClass: Codable {
+	// Properies need to be mutable, otherwise you get this warning:
```

Comment:
Alternatively make these `let` and don't assign them initial values? Having them be `var` values is fine though, just not sure if your comment is a way of indicating you find this to be a workaround of some sort.

Reply:
Yup, I think this is fine as-is.